docs/static-pvc.md

```diff
@@ -0,0 +1,240 @@
+
+This document outlines how to use static PVC and static PV for rbd CSI driver
+
+**warning** static PVC can be created, deleted, mounted and unmounted but
+doesn't support other operations like (snapshot, resize, etc)
+
+## RBD static volume
+
+rbd images created manually can be mounted and unmounted to an app, below step
+shows how to create a static PV, static PVC and bind PVC to an application
+
+### Create static RBD Filesystem PV
+
+to create the rbd PV you need to know the `rbd image name`,`clusterID` and
+`pool` name in which the rbd image is created
+
+```yaml
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: fs-static-pv
+spec:
+  accessModes:
+  - ReadWriteOnce
+  capacity:
+    storage: 4Gi
+  csi:
+    driver: rbd.csi.ceph.com
+    fsType: ext4
+    nodeStageSecretRef:
+      # node stage secret name
+      name: rook-csi-rbd-node
+      # node stage secret namespace where above secret is created
+      namespace: rook-ceph
+    volumeAttributes:
+      # cluster id specified in configmap during csi deployment
+      clusterID: abcd
+      imageFeatures: layering
+      imageFormat: "2"
+      pool: replicapool
+      staticVolume: "true"
+    # name of the rbd image created
+    volumeHandle: rbd-test
+  persistentVolumeReclaimPolicy: Retain
+  volumeMode: Filesystem
+```
+
+```bash
+[$] kubectl create -f fs-static-pv.yaml
+persistentvolume/fs-static-pv created
+```
+
+### Create static RBD Filesystem PVC
+
+to create the rbd PVC you need to know the PV name which is created above
+
+```yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: fs-static-pvc
+  namespace: default
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 4Gi
+  volumeMode: Filesystem
+  # volumeName is static PV name
+  volumeName: fs-static-pv
+```
+
+```bash
+[$] kubectl create -f fs-static-pvc.yaml
+persistentvolumeclaim/fs-static-pvc created
+```
+
+now you can bind this PVC to any app pod which uses filesystem PVC
+
+### Create app pod to use the Filesystem PVC
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: csi-rbd-demo-pod
+spec:
+  containers:
+    - name: web-server
+      image: nginx
+      volumeMounts:
+        - name: mypvc
+          mountPath: /var/lib/www/html
+  volumes:
+    - name: mypvc
+      persistentVolumeClaim:
+        claimName: fs-static-pvc
+```
+
+```bash
+[$] kubectl create -f pod.yaml
+pod/csi-rbd-demo-pod created
+```
+
+wait for app pod to go to running state, now you can consume static PVC for
+storing data
+
+### Delete Resources created for static Filesystem PV
+
+```bash
+[$] kubectl delete pod/csi-rbd-demo-pod
+[$] kubectl delete pvc/fs-static-pvc
+[$] kubectl delete pc/fs-static-pv
+```
+
+*Note* deleting PV and PVC doesnot deleted the backend rbd image, user need to
+manually delete the rbd image if required
+
+### Create static RBD Block PV
+
+to create the rbd PV you need to know the `rbd image name`,`clusterID` and
```

Comment:
This section is mostly a repeat of the filesystem PV other than ` volumeMode: Block`, I would suggest just calling that part out and not repeating the entire process. Helps keeps the reading shorter and also future maintainability easier.